docs/interfaces/postgres.rst

```diff
@@ -251,13 +251,13 @@ is implemented::
     +-----------------------+
     SHOW 1 row in set (... sec)
 
-BEGIN/COMMIT statements
------------------------
+BEGIN/START/COMMIT statements
+-----------------------------
 
 For compatibility with clients that use the Postgres wire protocol, such as the
 Golang lib/pq and pgx drivers, the full PostgreSQL syntax of the
-:ref:`BEGIN <ref-begin>` and :ref:`COMMIT <ref-commit>` statements is
-implemented, for example::
+:ref:`BEGIN <ref-begin>`, :ref:`START <ref-sql-start-transaction>` and :ref:`COMMIT <ref-commit>`
+statements are implemented, for example::
```

Comment:
how about:

    For compatibility with clients that use the PostgresSQL wire protocol (e.g.,
    the Golang lib/pq and pgx drivers), CrateDB will accept the :ref:`BEGIN
    <ref-begin>`, :ref:`COMMIT <ref-commit>`, and :ref:`START TRASNACTION
    <sql-start-transaction>` statements. For example::

changes here:

- fixed "Postgres" (as above)
- tidied up the grammar/phrasing (I realize this is the text you are modifying, but while we're in here we may as well tidy it up a little) :)
- expanded ``START`` to ``START TRANSACTION`` so we're using the statement in full (better to be as explicit as possible)
- reordered the list of statements to match the list of statements that appear later down this sectiobn
- I reworded things slightly so that we say that CrateDB "accepts" the statements. I think "implements" and "supports" may confuse matters slightly because both of them _may_ give the impression that these statements are something more than a no-op